### PR TITLE
fix(forms): Don't show spinner arrows in number input fields on Firefox

### PR DESCRIPTION
**Pull request by iionly**
Fixes https://github.com/Elgg/Elgg/issues/10288.

### CODE REVIEW

File: install/css/install.css [hunk 1]
```diff
@@ -191,6 +191,10 @@ input[type="url"] {
 	width: 100%;
 }
 
+input[type="number"] {
```
Comment (mrclay) [inline install/css/install.css]: Should be `.elgg-input-number {`. We don't want to affect elements plugins may have added/styled manually.

Comment (mrclay) [inline install/css/install.css]: It was probably not good to do that with other inputs but we're stuck with them.

**Comment by iionly** [comment]
Don't merge yet. I forgot a file.

File: install/css/install.css (hunk 1 above)
Comment (iionly) [inline install/css/install.css]: I don't understand. If the style is changed for .elgg-input-number only you might end with differently styled number input fields if someone adds a input field not using the elgg-input-number class.

Comment (mrclay) [inline install/css/install.css]: Ignore my comment. As we're already styling bare elements like `input`, we can continue.

Ideally I prefer a framework to only style elements with prefixed classnames so plugins can add their own markup and style it as necessary without interference. But we're well beyond that.

**Comment by mrclay** [comment]
This is a **chore** as this feature hasn't been released.

**Review by mrclay — APPROVED** [review]
Should probably be a "chore" but I'm OK either way.

**Comment by iionly** [comment]
Made some mistake (don't know what exactly) in trying to add a second commit with the forgotten file and squashing them. Need to restart.

**Pull request closed without merging**